test.js

```diff
@@ -103,3 +103,19 @@ test('toBuf', function (t) {
 
   t.end()
 })
+
+test('toStr with path', function (t) {
+  t.plan(3)
+  var newKey = enc.toStr('3161616161616161616161616161616161616161616161616161616161616161/path')
+  t.equal(newKey, '3161616161616161616161616161616161616161616161616161616161616161')
+  t.equal(typeof newKey, 'string')
+  t.ok(enc.decode(newKey), 'valid key is now valid buf')
+})
+
+test('toStr with version', function (t) {
+  t.plan(3)
+  var newKey = enc.toStr('4161616161616161616161616161616161616161616161616161616161616161+5')
+  t.equal(newKey, '4161616161616161616161616161616161616161616161616161616161616161')
+  t.equal(typeof newKey, 'string')
```

Comment:
no need to check the type since the line above already checks for strict equality